uint8 nodeID
float32[3] position
string sub
bool on

